sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchClient.java

```diff
@@ -39,7 +39,7 @@
 
     /**
      * Package private constructor to be used by {@link SearchClientBuilder}
-     *
+     *classWithGettersWithAnnotations
```

Comment:
Was this intended? Seems like random string in the javadoc.

Reply:
Likely an accidental paste, will remove it.